WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -126,4 +126,10 @@ def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
     date_range(from_date, to_date, options)
   end
+
+  def alert(type, content=nil, note: false, &block)
```

Comment:
I actually almost suggested `bs_alert` for the same reason, but decided it wasn't that big of a deal. If I had known someone else would say the same thing, I probably would have aksed @jonatanklosko to change it =)

I think the content as a non keyword argument fits with other rails methods such as (content_tag](http://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag), and saves a lot of repeated `content: ...` for users.

I'm always for more tests =). Our view code is pretty lightly tested, however (I only write a test once something starts to feel like a hotspot).

I think people are going to discover this method from seeing it being used, not from reading our code and finding the method, but I'm not opposed to documentation here, as I'm hopeful the signature won't change (famous last words).

None of these feel critical to me, but I'm open to PRs addresing any of these. @jonatanklosko @larspetrus, feel free to take on any of these if you feel like they really speak to you =)